tests/show_run_bgp_test.py

```diff
@@ -0,0 +1,209 @@
+neighbor fc00::6 description ARISTA03T2
+
+"""
+
+class TestShowRunBgpSingleAsic(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        from .mock_tables import mock_single_asic
+        importlib.reload(mock_single_asic)
+        from .mock_tables import dbconnector
+        dbconnector.load_namespace_config()
+
+    @pytest.mark.parametrize('setup_single_bgp_instance',
+                             [
+			        'show_run_bgp',
+                             ],
+                             indirect=['setup_single_bgp_instance'])
+
+    def test_show_run_bgp_single(self,
+                                 setup_single_bgp_instance):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["runningconfiguration"].commands["bgp"], [])
+        print("{}".format(result.output))
+        assert result.exit_code == 0
+        assert result.output == show_run_bgp_sasic
+
+    @classmethod
+    def teardown_class(cls):
+        print("TEARDOWN")
+        os.environ['UTILITIES_UNIT_TESTING'] = "0"
+        from .mock_tables import mock_single_asic
+        importlib.reload(mock_single_asic)
+        from .mock_tables import dbconnector
+        dbconnector.load_database_config()
+
+
+class TestShowRunBgpMultiAsic(object):
+    @classmethod
+    def setup_class(cls):
+        print("SETUP")
+        from .mock_tables import mock_multi_asic
+        importlib.reload(mock_multi_asic)
+        from .mock_tables import dbconnector
+        dbconnector.load_namespace_config()
+
+    @pytest.mark.parametrize('setup_multi_asic_bgp_instance',
+                             [
+				'show_run_bgp',
+                             ],
+                             indirect=['setup_multi_asic_bgp_instance'])
+    def test_show_run_bgp_all_asics(self,
+                           setup_multi_asic_bgp_instance):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["runningconfiguration"].commands["bgp"], [])
+        print("{}".format(result.output))
+        assert result.exit_code == 0
+        assert result.output == show_run_bgp_masic
+
+
+    @pytest.mark.parametrize('setup_multi_asic_bgp_instance',
+                             [
+				'show_run_bgp',
+                             ],
+                             indirect=['setup_multi_asic_bgp_instance'])
+
+    def test_show_run_bgp_asic0(self,
+                                setup_multi_asic_bgp_instance):
+        runner = CliRunner()
+        result = runner.invoke(show.cli.commands["runningconfiguration"].commands["bgp"], ["-nasic0"])
+        print("{}".format(result.output))
+        assert result.exit_code == 0
+        assert result.output == show_run_bgp_masic_asic0
+
+    @classmethod
+    def teardown_class(cls):
+        print("TEARDOWN")
+        from .mock_tables import mock_single_asic
+        importlib.reload(mock_single_asic)
+        from .mock_tables import dbconnector
+        dbconnector.load_database_config
```

Comment:
can you add some tests for negative tests like bgp container is not running.

Reply:
Hi @arlakshm , added neg UT